README.adoc

```diff
@@ -62,7 +62,7 @@ Minishift is an open-source project dedicated to developing and supporting Minis
 The code base is forked from the https://github.com/kubernetes/minikube[Minikube] project.
 
 Contributions, questions, and comments are all welcomed and encouraged!
-Minishift developers hang out on IRC in the #minishift channel on Freenode.
+Join the Minishift team on the #minishift IRC channel on  https://freenode.net/[Freenode] or sign up for the https://lists.minishift.io/admin/lists/minishift.lists.minishift.io[Minishift Mailing List].
```

Comment:
We should make the information about IRC channel and mailing list more prominent in the readme. e.g.

**Join the Minishift community at**

*  #minishift IRC channel on [Freenode](https://freenode.net/)
* [Minishift Mailing List](https://lists.minishift.io/admin/lists/minishift.lists.minishift.io)